### PR TITLE
f-added homepage to quiz

### PR DESCRIPTION
**Pull request by surya-nemiwal**
created homepage for quiz webapp and added a cart to it.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
+:root{
+    --color-1 : rgb(33, 37, 41);
+    --color-2 : rgb(52, 58, 64);
+    --color-3 : rgb(73, 80, 87);
+    --color-4 : rgb(108, 117, 125);
+    --color-5 : rgb(206, 212, 218);
+    --color-6 : rgb(233, 236, 239);
```
Comment (bharati-21) [inline style.css:7]: Variable naming could be better here. You can use words like- primary, secondary, accent, border, background etc. Much more understandable.

**Review by bharati-21 — COMMENTED** [review]
Looks good. but dropped a few suggestions.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (sonishreyas) [inline index.html:6]: you can add a meta author tag here

File: style.css [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+body{
+    margin: 0rem;
+    text-align: center;
+    font-family: 'Quicksand', sans-serif;
+    background-color: var(--color-1);
+}
+
+header{
+    background-color: var(--color-2);
+    padding: .5rem 0rem 1rem;
```
Comment (sonishreyas) [inline style.css:19]: margin and paddings can be put in as a variable to maintain consistent styling

File: style.css [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    color: var(--color-5);
+    text-align: center;
+    margin: 0rem;
+}
+
+.card{
+    text-align: center;
+    background-color: var(--color-4);
+    color: black;
+    width: 300px;
```
Comment (sonishreyas) [inline style.css:32]: This can be changed to rem

**Review by sonishreyas — APPROVED** [review]
Have kept a few suggestions, which can be done later. Rest looks good to merge

**Pull request merged**